sw/vendor/riscv-opcodes/inst.rs

```diff
@@ -0,0 +1,2178 @@
+// Automatically generated by parse-opcodes.
```

Comment:
Didn't @fabianschuiki mention that this shouldn't be checked in in #61?

Reply:
Good point! That's probably a left-over thing in this branch. Maybe a rebase will do?

Reply:
Too many feature branches, sorry. Should be resolved now